api/optimeet/groups/urls.py

```diff
@@ -5,5 +5,8 @@
 
 urlpatterns = [
     path("", views.groups),
-    path("<int:group_id>/", views.group_detail)
+    path("<int:group_id>/", views.group_detail),
```

Comment:
Let's change the `int` type to `str` in order to remain consistent with the types, since `group_id` is a `CharField` in the models and DB.